File: problems/w4/shorter-than-5.py
# Implement the function shorter_than_5(strings), which returns a list that contains all of the strings from the strings input that have a length less than 5.

def shorter_than_5(strings):
    results = []

    # your code here
    for i in strings:
        if len(i) < 5:
            results.append(i)


    return results

File: problems/w4/shorter-than-5.js
// Implement the function shorterThan5(strings), which returns an array that contains all of the strings from the strings input that have a length less than 5.

const shorterThan5 = function(strings) {
    const results = [];

    // your code here
    for (let i of strings) {
        if (i.length < 5) {
            results.push(i);
        }
    }

    return results;
};
